diff -Naur nautilus-3.14.0.orig/src/nautilus-view.c nautilus-3.14.0/src/nautilus-view.c
--- nautilus-3.14.0.orig/src/nautilus-view.c	2014-09-02 03:33:00.000000000 +0700
+++ nautilus-3.14.0/src/nautilus-view.c	2014-09-26 13:00:16.789874939 +0700
@@ -7208,7 +7208,7 @@
   /* tooltip */                  NULL,
 				 G_CALLBACK (action_rename_select_all_callback) },
   /* name, stock id */         { NAUTILUS_ACTION_TRASH, NULL,
-  /* label, accelerator */       N_("Mo_ve to Trash"), "<control>Delete",
+  /* label, accelerator */       N_("Mo_ve to Trash"), "Delete",
   /* tooltip */                  N_("Move each selected item to the Trash"),
 				 G_CALLBACK (action_trash_callback) },
   /* name, stock id */         { NAUTILUS_ACTION_DELETE, NULL,
@@ -9851,9 +9851,9 @@
 	g_object_class_install_properties (oclass, NUM_PROPERTIES, properties);
 
 	binding_set = gtk_binding_set_by_class (klass);
-	gtk_binding_entry_add_signal (binding_set, GDK_KEY_Delete, GDK_CONTROL_MASK,
+	gtk_binding_entry_add_signal (binding_set, GDK_KEY_Delete, 0,
 				      "trash", 0);
-	gtk_binding_entry_add_signal (binding_set, GDK_KEY_KP_Delete, GDK_CONTROL_MASK,
+	gtk_binding_entry_add_signal (binding_set, GDK_KEY_KP_Delete, 0,
 				      "trash", 0);
 	gtk_binding_entry_add_signal (binding_set, GDK_KEY_KP_Delete, GDK_SHIFT_MASK,
 				      "delete", 0);
